src/typeahead/typeahead.directive.ts

```diff
@@ -27,7 +27,7 @@ import { TypeaheadConfig } from './typeahead.config';
 
 // eslint-disable-next-line
 type TypeaheadOption = string | Record<string | number, any>;
-type TypeaheadOptionArr = TypeaheadOption[] | Observable<TypeaheadOption[]>;
+type TypeaheadOptionArr = TypeaheadOption[] | Observable<TypeaheadOption>;
```

Comment:
@yuri-apanasik @valorkin  that's just sweeping the issue under the rug instead of addressing the actual problem.
Most likely wrong rxjs function is used on line `485`: it should be `of()` instead of `from()`.

Reply:
I would say 'from()' is correct there according to existing logic, but up to lib authors to make final decision on this question

Reply:
I would say 'from()' is correct there according to existing logic, but up to lib authors to make final decision on this question